### ISSUE TITLE
Explain how to generate the web pages

### ISSUE THREAD

**Issue by ttekampe** [lhcb/first-analysis-steps#196]
What do I need to do to make my newly contributed .md files show up in the Starterkit website?

**Comment by betatim** [comment]
Do the instructions in https://github.com/lhcb/first-analysis-steps/blob/master/CONTRIBUTING.md still work? After installing the dependencies `make preview` should take care of everything. For the "live" webpage a robot does everything.

**Comment by ttekampe** [comment]
This does not work for me. I have a newer version of commonmark (0.7.1-1) installed on my system and pip tells me that it does not find `CommonMark==0.5.4`

```
  Could not find a version that satisfies the requirement requirements.txt (from versions: )
No matching distribution found for requirements.txt
```

Unfortunately there was a lot of change in the package itself.
DocParser was renamed to Parser and the container it uses internally to store data has changed.

**Comment by betatim** [comment]
`virtualenv` to the rescue?

**Comment by cmarinbe** [comment]
This is solved by #202, right? Should we close this?

**Comment by ttekampe** [comment]
yes I think so.
